automation/run-tests.sh

```diff
@@ -70,6 +70,13 @@ function install_nmstate {
     '
 }
 
+function mark_eth1_eth2_nm_managed {
```

Comment:
You closed #558 because you wrote it was not needed anymore. But it is still here... Did you maybe miss this? Nevertheless, IMHO the config file that makes them unmanaged should be removed instead, see #568

Reply:
Thanks.
Now I know why I cannot reproduce the problem(I forgot the remove the fix from my branch as that was hidden in a commit).
I will reopen #558 with more detail on root cause of it.